src/main/java/com/rtb/projectmanagementtool/user/CreateUserServlet.java

```diff
@@ -0,0 +1,60 @@
+// incomplete
+// build other servlets first
+package com.rtb.projectmanagementtool.user;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import com.google.appengine.api.users.*;
+import com.rtb.projectmanagementtool.auth.*;
+import java.io.*;
+import java.util.*;
+import javax.servlet.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.*;
+
+/** Servlet that returns task data */
+@WebServlet("/create-user")
+public class CreateUserServlet extends HttpServlet {
+
+  DatastoreService datastore;
+  Cookie cookie;
+
+  public CreateUserServlet() {
+    datastore = DatastoreServiceFactory.getDatastoreService();
+  }
+
+  // for testing
+  public CreateUserServlet(DatastoreService datastore) {
+    this.datastore = datastore;
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // new AuthOps object
+    AuthOps auth = new AuthOps(datastore);
```

Comment:
Good steps so far.  Probably doable at this point to also get the form data from the parameters and set them into t he blank UserData before pushing into datastore